php/libraries/Utility.class.inc

```diff
@@ -1020,4 +1020,41 @@ class Utility
         }
         return $formData;
     }
+
+    /**
+     * Update the candidate_diagnosis_evolution_rel diagnosis confirmation status
+     *
+     * @param SessionID $sessionID  The sessionID that has been approved
+     * @param bool      $isApproved Specifies if session approval is set to pass
+     *
+     * @return void
+     */
+    static function updateDiagnosisConfirmationStatus(
+        SessionID $sessionID,
+        bool $isApproved
+    ) : void {
+        $factory = \NDB_Factory::singleton();
+        $DB      = $factory->database();
+        $status  = $isApproved ? 'Y' : 'N';
+
+        $dxToUpdate = $DB->pselectRow(
+            "SELECT cde.CandID, cde.DxEvolutionID
+            FROM candidate_diagnosis_evolution_rel cde 
+            JOIN diagnosis_evolution de USING (DxEvolutionID)
+            JOIN session s ON (cde.CandID=s.CandID AND s.Visit_label=de.visitLabel)
+            WHERE s.ID=:sid",
+            ['sid' => $sessionID]
+        );
+
+        foreach ($dxToUpdate as $key => $row) {
```

Comment:
```suggestion
        foreach ($dxToUpdate as $row) {
```
php/libraries/Utility.class.inc:1049 PhanUnusedVariable Unused definition of variable $key